docs/examples/activity-panel-inbox.md

```diff
@@ -234,7 +234,11 @@ function wapi_example_two_deactivate() {
 register_deactivation_hook( __FILE__, 'wapi_example_two_deactivate' );
 ```
 
+#### Using the REST API
+
+A limited set of note fields can be updated over the REST API: `status` and `date_reminder`.
+
 ## Questions?
 
 This is just the tip of the iceberg for possibilities for your own extensions to WooCommerce. Check
-out the new [sales record notes](https://github.com/woocommerce/woocommerce-admin/blob/master/includes/class-wc-admin-notes-new-sales-record.php) and the [settings notes](https://github.com/woocommerce/woocommerce-admin/blob/master/includes/class-wc-admin-notes-settings-notes.php) for in the woocommerce-admin code itself for other examples of working with this fun new feature.
+out the new [sales record notes](https://github.com/woocommerce/woocommerce-admin/blob/master/includes/notes/class-wc-admin-notes-new-sales-record.php) and the [settings notes](https://github.com/woocommerce/woocommerce-admin/blob/master/includes/notes/class-wc-admin-notes-settings-notes.php) for in the woocommerce-admin code itself for other examples of working with this fun new feature.
```

Comment:
>for in the woocommerce-admin code itself for other examples of working with this fun new feature.

Might be reading this wrong, but I think this should be rewritten as:

>used in the woocommerce-admin code itself for other examples of working with this fun new feature.

Reply:
I think we just need to omit the first "for".